types/blocked/index.d.ts

```diff
@@ -19,6 +19,7 @@ declare function Blocked(callback: (ms: number) => void, options?: Blocked.Optio
 
 declare namespace Blocked {
   interface Options {
-    threshold: number;  // in milliseconds
+    threshold?: number;  // in milliseconds
```

Comment:
This is not required, it defaults to a value if it is not provided.
https://github.com/tj/node-blocked/blob/68dd44bfc7e69d93ce6de943599c2674522eec11/index.js#L6